src/main/resources/db/migration/V1_3__create_table_outgoing_dialogmelding.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE DIALOGMELDING_OUT
+(
+    id                              SERIAL PRIMARY KEY,
+    uuid                            CHAR(36) NOT NULL UNIQUE,
+    created_at                      TIMESTAMPTZ NOT NULL,
+    personident                     VARCHAR(11) NOT NULL,
+    behandler_ref                   CHAR(36), -- TODO: Kan være null?
```

Comment:
Tror ikke denne kan være null - må ha en behandlerRef for å vite hvem som skal motta dialogmelding